components/brave_shields/common/features.cc

```diff
@@ -29,6 +29,10 @@ const base::Feature kBraveAdblockCollapseBlockedElements{
 // always-on list, overriding any locally set preference.
 const base::Feature kBraveAdblockCookieListDefault{
     "BraveAdblockCookieListDefault", base::FEATURE_DISABLED_BY_DEFAULT};
+// When enabled, Brave will display a bubble inviting the user to turn on the
+// "Easylist-Cookie List" filter.
+const base::Feature kBraveAdblockCookieListOptIn{
+    "BraveAdblockCookieListOptIn", base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
Would be good to have this disabled by default.  We can enable via griffin, but would be good to not have it start running on folks machines on merge, so that we can coordinate with PR and across platforms

Reply:
Done.